### DISCUSSION TITLE
USBSID project ~ probable collab question?

### DISCUSSION THREAD

**Discussion by LouDnl** [frntc/SIDKick-pico#51] [General]
Hey frntc,

Great project, really loving it!
I'm working on a hobby project that interfaces any SID with a computer (linux only for now) through USB. This is not a reboot of SIDBlaster or something like that. I have everything working with Vice (custom adaptation) and SIDBerry (my adaptation). 
But I want to get the config tool working in Vice except it just wont load. I can get it to load if I decompress it and load it as PRG but it wont load any settings. I'm probably overlooking something.
I can see it trying to load something when I run the SYS commands, its just stops tho.

By the way, SkPico works better with the new firmware you released today!

Thanks in advance!
LouD
Ps. Is there a way to contact you outside of Git?

**Comment by frntc** [comment]
My mail address is in every source file, feel free to contact me this way ;-)

**Reply by LouDnl** [reply]
Thanks, saw that, asking first is nicer. Will send you an email!

**Reply by LouDnl** [reply]
Mail sent :+1:

**Comment by LouDnl** [comment]
Working great on the new firmware btw (even without config).
https://mastodon.social/@loudfpv/112543645607149109
https://x.com/retroLouDnl/status/1796992718159114486?t=fUXDb0N9Rgn-eMte27m8nQ&s=19
